Example/Sources/SampleData.swift

```diff
@@ -70,7 +72,7 @@ final class SampleData {
 
     var now = Date()
 
-    let messageTypes = ["Text", "Text", "Text", "AttributedText", "Photo", "Video", "Location", "Emoji"]
+    var messageTypes = ["text", "attributedText", "photo", "video", "location", "emoji"]
```

Comment:
I think the easiest way to do this would be to populate this array based on the values from `NSUserDefaults`

Reply:
I made some changes, reverting all modifications to Core. Let me know :)